esque/controller/consumergroup_controller.py

```diff
@@ -95,13 +95,13 @@ def create_consumer_group_offset_change_plan(
             consumer_id=consumer_id, topic_name_expression=topic_name
         )
         if consumer_group_state == "Dead":
-            self._logger.error("The consumer group {consumer_id} does not exist.")
+            self._logger.error(f"The consumer group {consumer_id} does not exist.")
             return None
         if consumer_group_state != "Empty":
             self._logger.error(
-                f"Consumergroup {consumer_id} is not empty. Setting offsets is only allowed for empty consumer groups."
+                f"Consumergroup {consumer_id} has active consumers. Please scale your consumers down and try again."
```

Comment:
```suggestion
                f"Consumergroup {consumer_id} has active consumers. Please turn off all consumers in this group and try again."
```
I think the word "scale" might be too k8s specific :slightly_smiling_face:

Reply:
Hahah oh yeah I didn't even think about that 😂